website/src/models/user.js

```diff
@@ -27,7 +27,7 @@ export let schema = new Schema({
     facebook: Schema.Types.Mixed, // TODO validate
     local: {
       email: String,
-      hashed_password: String,
+      hashed_password: String, // eslint-disable-line camelcase
```

Comment:
For https://github.com/HabitRPG/habitrpg/issues/6193, we should convert the password key to hashedPassword to conform to the camelcase rule.